### PR TITLE
added fixq/update for bad questions

### LINKED ISSUE #2: Add action with button
- Student receives notification
- All buttons from TA side should disappear for the student except for "Remove"
- Student can not be helped until notification is addressed (presses "Ok")

### PR DESCRIPTION
**Pull request by baozix**
fixes #2 
To test this, you must reset your database since a new 'blocked' field has been added.

Added block field to entries
Does not have update question functionality
Student receives notif when TA hits fix question button, which blocks student from being helped until they hit the update question button. They will still move up in the queue as entries above finish.
Screen Shot 2020-12-28 at 9 43 00 PM
Screen Shot 2020-12-28 at 9 43 36 PM
Screen Shot 2020-12-28 at 9 48 38 PM
Screen Shot 2020-12-28 at 9 55 00 PM

### CODE REVIEW

**Comment by baozix** [comment]
todo: needs testing with multiple active TAs and multiple students

File: static/js/client.js [hunk 1]
```diff
@@ -284,6 +290,52 @@ socket.on("add", function(message) {
...
+    });
+});
+
+socket.on("update", function(message) {
+    if (disable_updates) return;
+    checkAndUpdateSeq(message.seq);
+    
+    $("#queue li").each(function(index, item) {
+        if ($(item).data("entryId") == message.id) {
+            $(item).find("button").addClass("hide");
```
Comment (angelaz1) [inline static/js/client.js:327]: helping-text isn't cleared once student updates their question. Need to also clear helping-text here

Comment (baozix) [reply static/js/client.js:327]: fixed!

**Review by angelaz1 — APPROVED** [review]
Looks good :)

**Pull request merged**